OpenVBX/views/messages/details.php

```diff
@@ -124,6 +124,43 @@
 			<h3>Transcription</h3>
 			<?php endif; ?>
 			<div class="message-transcript"><?php echo (is_null($message->content_text) ? "(no transcription)" : $message->content_text) ?></div>
+			<?php if(substr($message->call_sid,0,2) == 'MM'):
+				echo "<br><br><h3>Media Attachements</h3>";
+				
+				$ci = & get_instance();
+				
+				require_once(APPPATH . 'libraries/twilio.php');
+				
+				$ci->twilio = new TwilioRestClient($ci->twilio_sid,$ci->twilio_token,$ci->twilio_endpoint);
+				
+				$media_url = "Accounts/{$this->twilio_sid}/Messages/{$message->call_sid}/Media.xml";
+				
+				$mediaObj = $ci->twilio->request($media_url, "GET");
+				
+				$media_xml = $mediaObj->ResponseXml;
+				
+				foreach($media_xml->MediaList->Media as $media):
+				
+					$media_uri = 'https://api.twilio.com/2010-04-01/Accounts/'.$this->twilio_sid.'/Messages/'.$message->call_sid.'/Media/'.$media->Sid;
+					
+					if(strpos($media->ContentType,'image')!==false):
+						echo '<img src="'.$media_uri.'" type="'.$media->ContentType.'">';
```

Comment:
Image elements don't have a `type` attribute. We should parse out the content type to build the appropriate url. This is something that can be done in the controller to keep all logic away from the view.